pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -35,6 +35,18 @@ import (
 	compute "google.golang.org/api/compute/v1"
 )
 
+// L7LoadBalancerSrcRanges contains the ranges of source ips used by the GCE L7 load balancer
+// for proxying client requests and performing health checks.
+func L7LoadBalancerSrcRanges() []string {
+	return []string{"130.211.0.0/22", "35.191.0.0/16"}
```

Comment:
make these flags with the default value as these so it can be changed w/out recompile